src/components/TextGenerationForm.tsx

```diff
@@ -201,10 +215,11 @@ function TextGenerationForm(): JSX.Element {
             render={({ field: { onChange, value } }) => (
               <Switch.Root
                 className="SwitchRoot"
+                id="emoji"
                 checked={value}
                 onCheckedChange={onChange}
               >
-                <Switch.Thumb className="SwitchThumb" />
+                <Switch.Thumb tabIndex={0} className="SwitchThumb" />
```

Comment:
Right now tab keys focus on two things: `Switch.Root` and `Switch.Thumb`. I think `Switch.Root` is set to be focusable by default, but it does not has a visible border. Can you make the border visible and remove the `tabIndex` on `Swicth.Thumb`? thx